_minutes/2023-09-12-wecg-tpac.md

```diff
@@ -0,0 +1,242 @@
+ * [oliver] Also worth noting that this isn't something we'd want to address in the group, but we can optimize this process in the store to improve review times for these cases.
+ * [oliver] So the safe rules proposal is basically working on the basis about some rules being safe.  You might be getting close to the limit and want to add more dynamic rules.  [Proposal](https://docs.google.com/document/d/1srkkCJkl4X2KOOUwnpDd-kvm8AY70eZBAYI4m836bvQ/edit#heading=h.bz47t7liilts) shared in [issue 319](https://github.com/w3c/webextensions/issues/319) (“Increase the maximum number of dynamic rules”), add safe and unsafe categories (naming to be finalized).  The simplest division would be saying "block and allow rules are safe".  We did some research, AdGuard shared that 98 - 99% of their rules fall into those categories (in issue 319).  We could raise the limits without risking the concerns with dynamic rules we mentioned earlier.  Any thoughts?  Safari, Firefox, thoughts?  Would we want to have this across-browser?
+ * [rob] From the Firefox side, I'm supportive of the functionality, but I'm concerned about scoping the work being done. The original proposal included allowAllRequests which isn't required but not free to support.
+ * [oliver] The proposal does call out that we could potentially expand the definition in the future (e.g., CSP restrictions are probably safe, but more nuanced)
+ * [tomislav] I think it makes sense to divide between what you expect is safe and what you need to scrutinize more closely.
+ * [oliver] Timothy?
+ * [timothy] I'll need to read the proposal more in detail, but seems reasonable to me
+ * [oliver] Cool, I think we can continue the discussion on the thread
+ * [devlin] Time check: 5 minutes
+ * [ioliver] What are wish list items for DNR?
+ * [tomislav] Cosmetic rules is a popular request. “Safe CSS” such as hiding elements may be safe to do. That does obviously not fit with the name of the DNR API. Bikeshedding aside, would that be something fitting in DNR?
+ * [devlin] I'm still concerned about grouping these with Declarative Net Requests. I'm supportive of declarative style rules, but I don't think it belongs in that namespace.
+ * [rob] Would suggest focusing our remaining time on use cases.
+ * [sam macbeth] General concern with DNR development going forward.  Feature detection of what is available.  If a static rule fails (depending on the failure), it's a hard failure.
+ * [rob] Is it a hard failure?
+ * [sam macbeth] Rule failure causes the list to not load.
+ * [rob] In firefox, it's a soft failure (invalid static rules are ignored). Invalid dynamic/session rules result in a rejection.
+ * [oliver] I don't think this has come up in the community group before. We should probably create an issue to track.
```

Comment:
- [ ] File issue about handling invalid DNR rules (primarily static rules, dynamic/session rules can be mentioned for completeness)

Reply:
Filed an issue: https://github.com/w3c/webextensions/issues/449